instrument/Facilities.xml

```diff
@@ -740,19 +733,15 @@
   </instrument>
 
   <instrument name="PANTHER">
-    <technique>Neutron Spectroscopy</technique>
-    <technique>Reactor Direct Geometry Spectroscopy</technique>
-    <technique>Disk chopper time-of-flight spectrometer PANTHER</technique>
+    <technique>Spectroscopy</technique>
   </instrument>
 
   <instrument name="SHARP">
-    <technique>Neutron Spectroscopy</technique>
-    <technique>Reactor Direct Geometry Spectroscopy</technique>
-    <technique>Disk chopper time-of-flight spectrometer</technique>
+    <technique>Spectroscopy</technique>
   </instrument>
 
   <instrument name="D7">
-    <technique>Polarized diffraction</technique>
+    <technique>Polarized Diffraction</technique>
```

Comment:
For consistency with diffractometers, would you agree that should these technique names be written as a single word?

Reply:
Seems good I guess